features/cellular/UNITTESTS/at/at_cellularnetwork/test_at_cellularnetwork.cpp

```diff
@@ -76,12 +76,12 @@ class my_AT_CN : public AT_CellularNetwork {
             return _stack;
         }
     }
-    virtual bool has_registration(RegistrationType reg_type)
+    virtual int has_registration(RegistrationType reg_type)
     {
         if (reg_type == C_GREG) {
-            return false;
+            return 0;
         }
-        return true;
+        return 2;
```

Comment:
"#define" for numbers 0 and 2 would help reading for people unfamiliar with Cellular

Reply:
I did not use define because the value comes from 3GPP spec and you need to read that in any case to understand what you are doing. Also when the mode is a define and a modem specific porting requires a new mode they also need to make a define also in the generic codes at AT layer. But I can change that if you think its more clear with define?

Reply:
I'd like to echo the sentiment of no magic numbers where possible, even if they are hardcoded in some spec.

Reply:
#define works in this case as a hint, it might be the case that developer has read the spec and with this hint s/he finds the context immediately without reading the spec again. Let's define those.

Reply:
@AnttiKauppila please review